node/src/components/linear_chain_sync/peers.rs

```diff
@@ -0,0 +1,91 @@
+use std::collections::VecDeque;
```

Comment:
I think the entire `linear_chain_sync` component has been removed on the `feat-fast-sync` branch.